src/CalcViewModel/Common/TraceLogger.cpp

```diff
@@ -379,9 +389,86 @@ namespace CalculatorApp
             return;
 
         LoggingFields fields{};
-        fields.AddGuid(L"SessionGuid", sessionGuid);
-        fields.AddString(L"Mode", NavCategory::GetFriendlyName(mode)->Data());
+        fields.AddGuid(SESSION_GUID, sessionGuid);
+        fields.AddString(CALC_MODE, NavCategory::GetFriendlyName(mode)->Data());
         fields.AddUInt64(PDT_PRIVACY_DATA_TAG, PDT_PRODUCT_AND_SERVICE_USAGE);
         LogLevel2Event(EVENT_NAME_INPUT_PASTED, fields);
     }
+
+    void TraceLogger::LogShowHideButtonClicked(bool isHide)
```

Comment:
Parameter might better named as isHidden

Reply:
I updated it to isHIdeButton since this is logging if the Hide or Show function button clicked